backend/main.py

```
@@ -1,52 +1,49 @@
 
 
 
-from fastapi import FastAPI, Depends, HTTPException,Header
+from fastapi import FastAPI, Depends, HTTPException,Header, APIRouter
 from fastapi.security import OAuth2PasswordBearer
 from sqlalchemy.orm import Session
 from database import SessionLocal, engine
 from models import User
-from schemas import UserCreate, Token, LoginRequest, UserUpdate
+from schemas import UserCreate, Token, LoginRequest, UserUpdate, OrderOut, OrderCreate
```

Ruff (0.8.2)

`schemas.OrderOut` imported but unused

Remove unused import

(F401)